go.mod

```diff
@@ -5,6 +5,7 @@ go 1.12
 require (
 	github.com/emirpasic/gods v1.12.0
 	github.com/golang/mock v1.3.1
+	github.com/luci/go-render v0.0.0-20160219211803-9a04cc21af0f
```

Comment:
I can't see any code use it, could delete it?

Reply:
done